tests/data/dataset_readers/wikitables/wikitables_dataset_reader_test.py

```diff
@@ -16,11 +16,12 @@ def test_reader_reads(self):
         question_tokens = ["what", "was", "the", "last", "year", "where", "this", "team", "was",
                            "a", "part", "of", "the", "usl", "a", "-", "league", "?"]
         assert [t.text for t in instance.fields["question"].tokens] == question_tokens
-        assert actions == ['@@START@@', 'd', 'd -> [<d,d>, d]', '<d,d> -> M0', 'd -> [<e,d>, e]',
-                           '<e,d> -> [<<#1,#2>,<#2,#1>>, <d,e>]', '<<#1,#2>,<#2,#1>> -> R',
-                           '<d,e> -> D1', 'e -> [<r,e>, r]', '<r,e> -> [<<#1,#2>,<#2,#1>>, <e,r>]',
-                           '<<#1,#2>,<#2,#1>> -> R', '<e,r> -> C0', 'r -> [<e,r>, e]', '<e,r> -> C1',
-                           'e -> cell:usl_a_league', '@@END@@']
+        assert actions == ['@@START@@', 'd', 'd -> [<d,d>, d]', '<d,d> -> max', 'd -> [<e,d>, e]',
```

Comment:
Much improved!  Can you also do this with the types?  @OyvindTafjord had some questions around this - using single lower-case letters for the types means you can only ever have 26 types, right?

Reply:
You mean show types like `<<num,cell>,cell> -> [<cell,<<num,cell>,cell>>, cell]` instead of `<<d,e>,e> -> [<e,<<d,e>,e>>, e]`? That shouldn't be difficult, but I don't think it is necessary. We have 4-5 basic types in the domains we are looking at now, and I think keeping them as single lower-case letters makes it more readable. But yes, if we extend this to domains where there are a lot more types, that may be necessary, both for understanding what's happening and also for potentially having more than 26 basic types.

Reply:
Added a `TODO` for this.